bin/test-exercise

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Test an exercise
+
+# Move example files to where Cargo expects them
+cp "./src/lib.rs" "./lib-orig.rs"
+cp -f "./example.rs" "./src/lib.rs"
+if [ -f "./Cargo-example.toml" ]; then
+   cp "./Cargo.toml" "./Cargo-orig.toml"
+   cp -f "./Cargo-example.toml" "./Cargo.toml"
+fi
+cp -r "./tests" "./tests-orig"
+
+# eliminate #[ignore] lines from tests
+for test in "tests/*.rs"; do
+    sed -i '/\[ignore\]/d' $test
+done
+
+# run tests
+cargo test
+
+# reset
```

Comment:
optionally, all this reset business may move into a `trap` so that it will be run even if the script gets Ctrl-C in the middle of the `cargo test` run

I don't require it because perhaps that will be a rare occurrence.

One can see an example at https://github.com/exercism/go/blob/master/bin/test-without-stubs#L24-L39